spec/system/community/community_spec.rb

```diff
@@ -290,4 +290,32 @@
     expect(page).to have_text(topic_1.first_post.body)
     expect(page).to have_text(reply_1.body)
   end
+
+  scenario 'coach marks a post as solution, edits content, and checks last edited info' do
+    sign_in_user(coach.user, referer: topic_path(topic_1))
+
+    find("div[aria-label='Options for post #{reply_1.id}']").click
+    click_button 'Mark as solution'
+
+    # Refresh page and check that marking solution doesn't update last edited info
+    visit current_path
```

Comment:
Shouldn't we wait for the page to change in some way (by dismissing a notification, perhaps), before triggering the page reload?